client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.js

```diff
@@ -0,0 +1,14 @@
+angular.module('managerApp')
```

Comment:
This file act more as a `*.module.js` where you should import your state declaration from `*.routing.js`.

Reply:
???

Reply:
To bring more details :

- Rename this file `pack-xdsl-access-comfort-exchange.module.js`
- Export all the `$stateProvider.state` defintion in a `***.routing.js` file
- Import all your required files to make the module work (controller, routing etc...)